### PR TITLE
Cheapest Flights Within K Stops Graph Solution C++ Best Solution

### PR DESCRIPTION
**Pull request by Deepak-technical**

### CODE REVIEW

**Comment by Deepak-technical** [comment]
This is the most important Problem in Graphs Which has been asked by Google , Amazon, Microsoft etc. It is base on the famous Dijisktra Algorithm and it is a follow up of it

File: 17. Graph/Cheapest Flights Within K Stops.cpp [hunk 1]
```diff
@@ -0,0 +1,52 @@
+#include <bits/stdc++.h>
```
Comment (ritikbanger) [inline 17. Graph/Cheapest Flights Within K Stops.cpp:1]: Please add question as comment too. Also, change file name, refer to readme.

**Review by ritikbanger — CHANGES_REQUESTED** [review]
Some changes

File: 17. Graph/Cheapest Flights Within K Stops.cpp (hunk 1 above)
Comment (Deepak-technical) [reply 17. Graph/Cheapest Flights Within K Stops.cpp:1]: There are n cities connected by some number of flights. You are given an array flights where flights[i] = [fromi, to i, price i] indicates that there is a flight from city from i to city to i with cost price i.

You are also given three integers src, dst, and k, return the cheapest price from src to dst with at most k stops. If there is no such route, return -1.

**Comment by Deepak-technical** [comment]
what changes can you please specify

**Pull request closed without merging**